docs/visualize/tilemap.asciidoc

```diff
@@ -1,116 +1,16 @@
-+
-NOTE: Higher precisions increase memory usage for the browser that displays {kib} and the underlying
-{es} cluster.
-
-* *Place markers off grid (use {ref}/search-aggregations-metrics-geocentroid-aggregation.html[geocentroid])* &mdash; When you selected, the markers are
-placed in the center of all documents in the bucket, and a more accurate visualization is created. When deselected, the markers are placed in the center
-of the geohash grid cell.
-+
-NOTE: When you have multiple values in the geo_point, the coordinate map is unable to accurately calculate the geo_centroid.
-
-[float]
-[[navigate-coordinate-map]]
-==== Navigate the coordinate map
-
-To navigate the coordinate map, use the navigation options.
-
-* To move the map center, click and hold anywhere on the map and move the cursor.
-
-* To change the zoom level, click *Zoom In* or *Zoom out* image:images/viz-zoom.png[].
-
-* To automatically crop the map boundaries to the
-geohash buckets that have at least one result, click *Fit Data Bounds* image:images/viz-fit-bounds.png[].
-
-[float]
-[[region-map]]
-=== Region map
-
-Use region maps when you want to show statistical data on a geographic area, such as a county, country, province, or state. For example, use a region map if you want to see the average sales for each country with the sample eCommerce order data.
-
-[role="screenshot"]
-image::images/visualize_region_map_example.png[]
-
-[float]
-[[build-region-maps]]
-==== Build a region map
-
-Configure the `kibana.yml` settings and add the aggregations.
-
-. In `kibana.yml`, set `xpack.maps.showMapVisualizationTypes` to `true`.
-
-. To display your data on the region map, use the following aggregations:
-
-* <<visualize-metric-aggregations,Metric>>
-* <<visualize-sibling-pipeline-aggregations,Sibling pipeline>>
-* <<visualize-bucket-aggregations,Terms bucket aggregation>>
-
-[float]
-[[navigate-region-map]]
-==== Navigate the region map
-
-To navigate the region map, use the navigation options.
-
-* To change the zoom level, click *Zoom In* or *Zoom out* image:images/viz-zoom.png[].
-
-* To automatically crop the map boundaries, click *Fit Data Bounds* image:images/viz-fit-bounds.png[].
-
-[float]
 [[heat-map]]
-=== Heat map
+== Heat map
 
-Use heat maps when your data set includes categorical data. For example, use a heat map to see the flights of origin countries compared to destination countries using the sample flight data.
+Display graphical representations of data where the individual values are represented by colors. Use heat maps when your data set includes categorical data. For example, use a heat map to see the flights of origin countries compared to destination countries using the sample flight data.
 
 [role="screenshot"]
 image::images/visualize_heat_map_example.png[]
 
+NOTE: The maps in Visualize have been replaced with <<maps>>, which offers more functionality.
```

Comment:
Not sure if this is the right place for this note. Heat maps are very different from geospatial maps and the to should not be conflated.